src/box/lua/config/init.lua

```diff
@@ -212,8 +212,11 @@ function methods._collect(self, opts)
     end
 
     if cluster_config:find_instance(cconfig, self._instance_name) == nil then
+        local is_reload = self._status == 'reload_in_progress'
+        local action = is_reload and 'Reload' or 'Startup'
+        local source_info_str = table.concat(source_info, '\n')
         error(dedent([[
-            Startup failure.
+            %s failure.
```

Comment:
I guess the same is applicable for the error at line 196.

Reply:
Thank you for the review! I did not think that this error can be reproduced, but apparently I was mistaken. Fixed.